istio-telemetry/mixer-telemetry/templates/stack-driver.yaml

```diff
@@ -0,0 +1,820 @@
+{{- if .Values.mixer.adapters.stackdriver.enabled }}
```

Comment:
Could you rename the file to stackdriver.yaml?

Reply:
done.